openrtb-core/src/main/protobuf/openrtb.proto

```diff
@@ -1197,6 +1203,78 @@ message BidRequest {
     extensions 100 to 9999;
   }
 
+  // OpenRTB 2.6: Structured user agent information, which can be used when a client supports
+  // User-Agent Client Hints. If both device.ua and device.sua are present in the bid request,
+  // device.sua should be considered the more accurate representation of the device attributes.
+  // This is because the device.ua may contain a frozen or reduced user agent string
+  // due to deprecation of user agent strings by browsers.
+  message UserAgent {
+    // Each BrandVersion object identifies a browser
+    // or similar software component. Implementers should send brands and
+    // versions derived from the Sec-CH-UA-Full-Version-List header
+    // or an equivalent JavaScript accessor from NavigatorUAData interface.
+    // This header or accessor are only available for browsers that support User-Agent Client Hints.
+    // RECOMMENDED by the OpenRTB specification.
+    repeated BrandVersion browsers = 1;
+
+    // A BrandVersion object that identifies the
+    // user agent’s execution platform / OS. Implementers should send a brand
+    // derived from the Sec-CH-UA-Platform header, and version derived
+    // from the Sec-CH-UAPlatform-Version header
+    // or an equivalent JavaScript accessor from NavigatorUAData interface.
+    // This header or accessor are only available for browsers that support User-Agent Client Hints.
+    // RECOMMENDED by the OpenRTB specification.
+    optional BrandVersion platform = 2;
+
+    // 1 if the agent prefers a “mobile” version of the content, if available,
+    // i.e. optimized for small screens or touch input.
+    // 0 if the agent prefers the “desktop” or “full” content.
+    // Implementers should derive this value from the Sec-CH-UAMobile header
```

Comment:
thee MOST minor of comments
`Sec-CH-UAMobile` should be `Sec-CH-UA-Mobile`

Reply:
no, I think you're right. I see Sec-CH-UA-Mobile in the spec https://github.com/InteractiveAdvertisingBureau/openrtb2.x/blob/2.6-202211/2.6.md#3229---object-useragent-

Reply:
True, thats the name of the Header/CH
But looking at the IAB protobuf file they made a typo and we just inherited it lol
https://developers.google.com/authorized-buyers/rtb/downloads/openrtb-proto